src/Uno.UWP/Security.Credentials/PasswordVault.Android.cs

```diff
@@ -160,5 +164,37 @@ public override void Write(byte[] buffer, int offset, int count)
 					=> _adapter.Write(buffer, offset, count);
 			}
 		}
+
+		/// <summary>
+		/// Persister for devices bellow Android level 23.
+		/// RSA/ECB/PKCS1Padding only is supported and is not considered secure.
+		/// </summary>
+		private sealed class UnSecureKeyStorePersister : FilePersister
+		{
+			private const string _notSupported = @"RSA/ECB/PKCS1Padding with asymetric key is considered not secure and will not  be supported for device under API level 23";
+
+			private const string _lowLevelDeviceTransform = "RSA/ECB/PKCS1Padding";
+			private const string _provider = "AndroidKeyStore";
+			private const string _alias = "uno_passwordvault";
+			private readonly KeyPair _keyPair;
```

Comment:
`Security.Credentials/PasswordVault.Android.cs(179,29): error CS0169: The field 'PasswordVault.UnSecureKeyStorePersister._keyPair' is never used [/Users/runner/runners/2.163.1/work/1/s/src/Uno.UWP/Uno.csproj]`
```suggestion
```

Reply:
Actually all the fields except the `_notSupported` can be removed